src/main/java/com/useful/ucars/uCarsListener.java

```diff
@@ -111,15 +111,19 @@ public uCarsListener(ucars plugin) {
 	public Vector calculateCarStats(Entity car, Player player,
 			Vector velocity, double currentMult) {
 		if (UEntityMeta.hasMetadata(car, "car.frozen") || car.hasMetadata("car.frozen")) {
-			if(car.hasMetadata("inertialYAxis")) {
-				velocity = new Vector(0, velocity.getY(), 0); // Don't freeze Y-velocity
-			}
-			if(car.getType() == EntityType.MINECART) {
-				velocity = new Vector(0, GRAVITY_Y_VELOCITY_MAGNITUDE, 0);
-			} else {
-				velocity = new Vector(0, 0, 0);
+			if (UEntityMeta.hasMetadata(car, "car.frozen") || car.hasMetadata("car.frozen")) {
```

Comment:
You shouldn't need to duplicate the outer if statement here. Also might have been nice if the commit message was more descriptive - eg. "Fix inertial y axis velocity not being returned, bump to v24.1"